packages/custom-functions-metadata/src/parseTree.ts

```diff
@@ -708,6 +722,13 @@ function getParameters(parameterItem: IGetParametersArguments): IFunctionParamet
         type: ptype,
       };
 
+      // for backward compatibility, we put cell value type in cellValueType instead of type.
+      if (Object.values(CELLVALUETYPE_MAPPINGS).includes(ptype)) {
+        // @ts-ignore
```

Comment:
Is it possible to not have `@ts-ignore` here. Usually the code could be modified. Can you please look into this?